bundle.yaml

```diff
@@ -36,6 +36,21 @@ applications:
     channel: "stable"
     scale: 1
 
+  nginx-ingress-integrator-sdlc:
+    charm: "nginx-ingress-integrator"
+    channel: "stable"
+    scale: 1
```

Comment:
I think we need a `trust` attribute here? (and on the others...)

Reply:
Done